Common/include/geometry/dual_grid/CPoint.hpp

```diff
@@ -456,6 +456,18 @@ class CPoint {
   }
   inline void SetWall_Distance(unsigned long iPoint, su2double distance) { Wall_Distance(iPoint) = distance; }
 
+  /*!
+   * \brief Get the index of the closest wall element.
+   * \param[in] iPoint - Index of the point.
+   */
+  inline unsigned long GetClosestWall_Elem(unsigned long iPoint) {return ClosestWall_Elem(iPoint);}
```

Comment:
Can you add the param[out] for these please

Reply:
Sure, I'll do it now.